client/src/store/historyStore/model/filtering.js

```diff
@@ -191,6 +216,43 @@ export function getQueryDict(filterText) {
     return queryDict;
 }
 
+/** Returns a dictionary resembling filterSettings (for HistoryFilters):
+ * e.g.: Unlike getFilters or getQueryDict, this maintains "hid>":"3" instead
+ *       of changing it to "hid-gt":"3"
+ * Only used to sync filterSettings (in HistoryFilters)
+ * @param {String} filterText The raw filter text
+ */
+export function getFilterSettings(filterText) {
```

Comment:
This function basically serves the same function as `getFilters()`, except instead of parsing `filterText` to something like this: `"hid-gt":"3"`, it does this: `"hid>":"3"`. It basically converts `filterText` to a dict like `filterSettings` from https://github.com/galaxyproject/galaxy/blob/dev/client/src/components/History/CurrentHistory/HistoryFilters.vue